docs/migration_guide.rst

```diff
@@ -13,11 +15,24 @@ Migration guide
 
 0.23.0 Breaking changes
 -----------------------
+
 .. currentmodule:: starknet_py.utils.typed_data
 
 1. :class:`StarkNetDomain` has been renamed to :class:`Domain`
-
 2. :class:`TypedData` field ``domain`` has been changed from ``dict`` to :class:`Domain`
+3. :class:`BasicType` and :class:`PresetType` have been added
```

Comment:
Is it a breaking change?

Reply:
No, I moved it to minor changes section.